SVProgressHUD/SVProgressHUD.m

```diff
@@ -1181,5 +1155,62 @@ - (CGFloat)visibleKeyboardHeight{
     return 0;
 }
 
+#pragma mark - UIAppearance Setters
+
+- (void)setBackgroundColor:(UIColor *)color {
+    [super setBackgroundColor:[UIColor clearColor]];
+    // Check needed for UIAppearance to work (since UILabel uses setters in init)
```

Comment:
The comment is misleading. Why is UILabel mentioned. This is also true for all the following methods. I think you got UILabel from Peter Steinberger, because he used it for an UILabel subclass.